config/tf_modules/aws-eks/variables.tf

```diff
@@ -1,5 +1,20 @@
 data "aws_region" "current" {}
 
+locals {
+  user_data_prefix = <<EOT
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
+
+--==MYBOUNDARY==
+Content-Type: text/x-shellscript; charset="us-ascii"
+
+#!/bin/bash
```

Comment:
I feel like this should be specified by the user

Reply:
Nope, it MUST always be there or else it fails with an ugly error message

Reply:
Best just to have it there from the beginning as most users won't be clever enough to figure out that they need this-- support the golden path

Reply:
It’s a rare enough use case that customizability matters here more than golden path. So id say remove all of this and we can ask the user to specify the multi part stuff and bin/bash in the docs - that’s what aws docs do as well.